client/src/pages/PaymentDetail/PaymentDetail.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import { loadStripe } from '@stripe/stripe-js';
+import { Elements } from '@stripe/react-stripe-js';
+import SidePanel from '../../components/SidePanel/SidePanel';
+import StripeCard from './StripeCard/StripeCard';
+import useStyles from './useStyles';
+
+function PaymentDetail(): JSX.Element {
+  const classes = useStyles();
+
+  const stripePromise = loadStripe(
```

Comment:
Make sure to call `loadStripe` outside of a component’s render to avoid recreating the `Stripe` object on every render.

Ref: https://stripe.com/docs/stripe-js/react

Reply:
Sure. I will fix it.

Reply:
successfully implemented in commit 8340f128b930c04eb81e229095baa2623bda7942